.github/workflows/ci-base-tests-linux.yml

```diff
@@ -6,7 +6,7 @@ env:
   venv_dir: .venv
 
 jobs:
-  base-tests-linux:
+  base-tests:
```

Comment:
I am not against the change but is there any reason to change this?

Reply:
With the addition of more jobs in the same workflow, following the original naming style, each of the job would have been suffixed with `-linux`. The suffix appeared a little repetitive, given the workflow's name of `SMARTS CI Base Tests Linux` which indicates it is for Linux. In short, the change was for aesthetics. This is how the test results look like now.

![Tests](https://github.com/huawei-noah/SMARTS/assets/24203326/a385b2ff-3339-4a2e-a009-8245d334a464)